week-1-ts-ecosystem/w01e01-typescript-basics/src/user/userClass.ts

```diff
@@ -0,0 +1,15 @@
+import { userType } from "./userType";
+
+export class userClass{
```

Comment:
Nazwa klasy z dużej i bez słowa `Class` w nazwie. Nie ma takiej potrzeby.

Nie do końca rozumiem dlaczego ta zmiana kodu na klasę ?
Tam mogło to zostać tak jak do tej pory z funkcjami, jedynie żeby typowanie się zgadzało i odpowiedni podział w plikach.